src/ape/_cli.py

```diff
@@ -72,15 +72,22 @@ def _suggest_cmd(usage_error):
 
     @property
     def commands(self) -> Dict:
+        group_name = "ape_cli_subcommands"
         if not self._commands:
-            entry_points = metadata.entry_points()  # type: ignore
+            try:
+                entry_points = metadata.entry_points(group=group_name)  # type: ignore
+            except TypeError:
+                entry_points = metadata.entry_points()
+                entry_points = (
+                    entry_points[group_name] if group_name in entry_points else []  # type: ignore
+                )
 
-            if "ape_cli_subcommands" not in entry_points:
+            if not entry_points:
                 raise Abort("Missing registered cli subcommands")
 
             self._commands = {
-                clean_plugin_name(entry_point.name): entry_point.load
-                for entry_point in entry_points["ape_cli_subcommands"]
+                clean_plugin_name(entry_point.name): entry_point.load  # type: ignore
```

Comment:
There's a type error here now?

Reply:
The type error is being introduced here when trying to use the non-deprecated python 3.10 syntax.

Reply:
Since upgrading to python 3.10, I have just noticed a lot of weird things like this. i am trying to fix the ones I see.
Here is some more info on it: https://github.com/python/importlib_metadata/issues/298

Reply:
The mypy type erros is because `entry_points()` method returns either `SelectableGroups` (deprecated) or `EntryPoints`. It is supposed to always return `EntryPoints` when usings kwargs, i.e. `group=`. However, `mypy` does not realize that. Also, I tried importing the `EntryPoints` type but it is not declared in their `__all__` so I couldn't really. `type: ignore` seems to be the way until `SelectableGroups` is removed entirely.

Reply:
OKAY finally figured out what was going on for real - python 3.8 requires a separate library that works like the original way (dict stuff).